d2l-button.html

```diff
@@ -31,58 +34,51 @@
 				vertical-align: middle;
 				white-space: nowrap;
 				width: auto;
-				@apply(--d2l-button);
-				@apply(--d2l-button-clear-focus);
+				@apply --d2l-button;
+				@apply --d2l-button-clear-focus;
 			}
 			/* Firefox includes a hidden border which messes up button dimensions */
-			:host(::-moz-focus-inner) {
+			::slotted(::-moz-focus-inner) {
 				border: 0;
 			}
-			:host, :host([disabled]:hover), :host([disabled]:focus) {
+			::slotted(*), ::slotted([disabled]:hover), ::slotted([disabled]:focus) {
```

Comment:
I am curious what selector(s) `::slotted(*)` results in.  Generally, universal selector performs poorly.